### ISSUE TITLE
File organization feels arbitrary

### ISSUE THREAD

**Issue by eriknelson** [fusor/cpma#118]
It's not very clear where anything goes in the project. Pulling from my comment on another PR:

> It's not clear what should go into something like ocp/ocp.go file, for example. Especially when there is an ocp3 and an ocp4 pkg. This needs more thought. This is probably something that deserve's its own issue, and should be dealt with separately.

**Comment by gildub** [comment]
+1
This was more empiric than arbitrary but it does need a proper thinking.

**Comment by jmontleon** [comment]
@eriknelson do you feel we've done enough to close this issue?

**Comment by eriknelson** [comment]
@jmontleon I was considering this earlier, yeah I think it's making more sense after #128, closing.
